packages/mobile/src/screens/edit-track-screen/EditTrackScreen.tsx

```diff
@@ -164,6 +178,26 @@ const useEditTrackSchema = () => {
             message: errorMessages.previewStartZero,
             path: [TRACK_PREVIEW]
           }
+        )
+        .refine(
```

Comment:
should the check for min being greater than max happen here as well?

Reply:
No because this is just a single input!